apollo_msgs/ApollothirdPartyPerceptionRadarObstaclesradarObstacle[] radarobstacle
apollo_msgs/ApollocommonHeader apolloheader
int32 errorcode#ApollocommonErrorCode
